map/src/App.tsx

```diff
@@ -250,8 +268,8 @@ export default styled(App)`
         position: absolute;
         z-index: 100;
         max-width: 500px;
-        top: 10px;
-        left: 10px;
+        top: ${p => p.theme.spacingPx}px;
+        left: ${p => p.theme.spacingPx}px;
```

Comment:
Can you explain this function?

Can't you just put `theme.spacingPx` or something similar?

Reply:
no, p.theme is provided by the styled-components theme provider, p is the properties for the components.

Reply:
See: https://styled-components.com/docs/advanced

Reply:
I've never used styled components too much, just to do simple things.

Thanks for the explanation :smile: